README.md

```diff
@@ -78,8 +82,14 @@ making a new tag during step 2. This is essentially redeploying an older version
         - To: The phone number to send the final confirmation text to (i.e. your phone number)
         - From: Any number in the format +12223334444 (we don't use this number for anything, but Twilio will complain if it's missing or incorrectly formatted)
 
+## 8. Celebrate
```

Comment:
Love that you put this in!

Reply:
You had a good point that it was missing :smile: